src/config/dynamic-theme-fixes.config

```diff
@@ -20671,19 +20690,32 @@ a[data-testid] > svg {
     fill: ${goldenrod} !important;
 }
 #xwd-board rect[class^="xwd__cell--cell"] {
-    fill: ${darkgrey} !important;
+    fill: #595963 !important;
 }
 #xwd-board rect[class^="xwd__cell--highlighted"] {
-    fill: ${lightgrey} !important;
+    fill: #483F80 !important;
 }
 #xwd-board rect[class^="xwd__cell--selected"] {
-    fill: ${white} !important;
+    fill: #4679AA !important;
 }
 #xwd-board rect[class^="xwd__cell--related"] {
     fill: ${khaki} !important;
 }
 #xwd-board rect[class*="xwd__cell--penciled"] ~ text[text-anchor^="middle"] {
-    fill: ${blue} !important;
+    fill: ${white} !important;
+    opacity: 50% !important;
+}
+.xwd__cell--block {
+    fill: #171717 !important;
+}
+#xwd-board rect {
+    stroke: #171717 !important;
+}
+#xwd-board path {
+    stroke: #171717 !important;
+}
+#js-logo-nav > svg > rect {
+    fill: #000 !important;
 }
 
 IGNORE INLINE STYLE
```

Comment:
Add ti this section:
`#js-logo-nav > svg > rect` and the logo will work better in light/dark scheme.